kernel/src/net/socket/ip/stream/init.rs

```diff
@@ -103,6 +103,7 @@ impl InitStream {
 
     pub(super) fn init_pollee(&self, pollee: &Pollee) {
         pollee.reset_events();
-        pollee.add_events(IoEvents::OUT);
+        // Linux adds IoEvents::OUT and IoEvents::HUP for a newly created socket
```

Comment:
Perhaps it's slightly better to say `OUT and HUP events` or `` `IoEvents::OUT` and `IoEvents::HUP` ``.